build/html/docs/api_description.md

```diff
@@ -0,0 +1,136 @@
+    target="_blank">
+    Creating an assignment for an API client
+    <img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/share.svg"
+    height="12">
+</a>
+
+**Note:** Make sure to assign the appropriate  HPE GreenLake for bare metal user roles in HPE GreenLake Central.
+
+  + 3. Make a REST call to generate the API access token:
+
+```
+curl -i -X POST \
+  '{issuerUrl}/v1/token' \
+  -H 'Content-Type: application/x-www-form-urlencoded' \
+  -d 'client_id={clientID}' \
+  -d 'client_secret={clientSecret}' \
+  -d grant_type=client_credentials \
+  -d scope=hpe-tenant
+```
+Obtain the `access_token` from the response.
+
+<h2>Option2: Getting the token directly from UI</h2>
+
+To authenticate with the BMaaS API you need to obtain the access token from
+<a
+    href="https://client.greenlake.hpe.com"
+    target="_blank">
+    HPE GreenLake Central
+    <img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/share.svg"
+    height="12">
+</a>.
+Once logged in to **HPE GreenLake Central**,
+Click the
+<img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/user.svg">
+icon in the top right corner and then click
+<input
+    type="button"
+    value="API Access"
+    style="color: #000;
+        background-color: #fff;
+        border: 2px solid #00b388;
+        border-radius: 4px;"/>
+. In the API Access page, click the
+<img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/clipboard.svg">
+icon to copy the personal access token.
+
+<br>
+
+> <img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/circle-information.svg"> **INFO**
+For both the options, the access token lease expires in **15** minutes.
+
+## Project API Key
+The specific Project ID must be provided with **Project** header required when accessing service
+within a single project context. Used only when using HPE GreenLake IAM issued access tokens.
+
+## Membership API Key
+HPE Internal only. Required only when using classic Metal access tokens.
+
+# Getting a Space name or ID
+Space name or ID is required in header for few Project level APIs. You can obtain the Space name and the corresponding ID from 
+<a
+    href="https://client.greenlake.hpe.com"
+    target="_blank">
+    HPE GreenLake Central
+    <img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/share.svg"
+    height="12">
+</a>.   
+Once logged in to **HPE GreenLake Central**,
+Click the
+<img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/user.svg">
+icon in the top right corner and copy the value of **Space** field.  
+
+The corresponding ID for the Space can be extracted from the current URL in the address bar.  
+It is the value of **spaceid** query parameter at the end of a URL after a '?' symbol.
+
+# HPE GreenLake for bare metal API
```

Comment:
Does this go to the top of file?

Reply:
mike had the same comment. closing this and keeping the other conversation open.